adapters/eth_bool_test.go

```diff
@@ -0,0 +1,45 @@
+package adapters_test
+
+import (
+	"testing"
+
+	"github.com/smartcontractkit/chainlink/adapters"
+	"github.com/smartcontractkit/chainlink/internal/cltest"
+	"github.com/smartcontractkit/chainlink/store/models"
+	"github.com/stretchr/testify/assert"
+)
+
+var evmFalse = "0x0000000000000000000000000000000000000000000000000000000000000000"
+var evmTrue = "0x0000000000000000000000000000000000000000000000000000000000000001"
+
+func TestEthBool_Perform(t *testing.T) {
+	tests := []struct {
+		name     string
+		json     string
+		expected string
+	}{
+		{"bool false", `{"value":false}`, evmFalse},
```

Comment:
With the updated case statement that matches the story requirements, I feel like we can reduce the number of test cases. How do you feel about reducing it down to the following and adding one for when the `value` attribute does not exist?

```golang
{"value is bool false", `{"value":false}`, evmFalse},
{"value is bool true", `{"value":true}`, evmTrue},
{"value does not exist", `{}`, evmFalse},
```

Reply:
Sure thing I'll add those in today

Reply:
I checked the other adapters and none of them have a test for an empty json object. We'll need to add a check to make that test case pass.

Reply:
Let's make both the nonexistent value and null equate to `evmFalse`.

Also, good call on the other adapters not covering the nonexistent case. We should definitely add that.